src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsChannelMappingTest.java

```diff
@@ -286,4 +275,83 @@ void getHearingChannels_ifAdjourmentEnabled_getNextHearing() {
             .containsOnly(TELEPHONE);
     }
 
+    @DisplayName("When adjournment flag is false, returns hearing chanel from the case")
+    @Test
+    void getHearingChannels_ifAdjournmentEnabledAndNextHearingIsNull_returnDefaultHearingChannel() {
+        SscsCaseData caseData = SscsCaseData.builder()
+            .appeal(Appeal.builder()
+                .hearingOptions(HearingOptions.builder()
+                    .wantsToAttend(NO.getValue()).build())
+                        .appellant(Appellant.builder()
+                            .name(Name.builder()
+                                .title("title")
+                                .firstName("first")
+                                .lastName("last").build())
+                        .build())
+                    .build())
+            .dwpIsOfficerAttending(NO.getValue())
+            .adjournment(Adjournment.builder().typeOfNextHearing(null).build())
+            .build();
+
+        given(referenceDataServiceHolder.isAdjournmentFlagEnabled()).willReturn(true);
```

Comment:
This contradicts the name of the test

Reply:
Updated the test description as per the test case.